packages/svelte/src/store/shared/index.js

```diff
@@ -168,7 +170,11 @@ export function derived(stores, fn, initial_value) {
 			)
 		);
 		started = true;
-		sync();
+		if (active_effect === null) {
+			sync();
+		} else {
+			queue_micro_task(sync);
```

Comment:
This looks concerning - changing the timing of derived stores depending on surroundings looks wrong. Why was this needed? Does this hint at a deeper problem with that validation, or our stores?

Reply:
The test looks a bit suspect tbh, I see you added it 7 months ago too. Is the `count` mutation important to be inside the derived?

Reply:
IIRC it was to test how many times it was called. Untracking is fine for this test though I can't help but wonder if someone does this in real life. Only way to find out is to try I guess